### PR TITLE
Don't save the connection made from Azure pane

### PR DESCRIPTION
**Pull request by aaomidi**
https://github.com/microsoft/azuredatastudio/issues/10140

### CODE REVIEW

File: extensions/azurecore/src/azureResource/commands.ts [hunk 1]
```diff
@@ -231,8 +231,8 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 			return;
 		}
 		// Ensure connection is saved to the Connections list, then open connection dialog
-		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: true });
-		const conn = await appContext.apiWrapper.openConnectionDialog(undefined, connectionProfile, { saveConnection: true, showDashboard: true });
+		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: false });
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/azureResource/commands.ts:234]: Why do you consider this a bug? It seems like giving users a way to specifically add a connection to the Servers view from the Azure view is useful to have and this action should only happen if they specifically click the "Connect" action.

File: extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export class AzureResourceDatabaseTreeDataProvider extends ResourceTreeDataProvi
 				databaseName: database.name,
 				userName: database.loginName,
 				password: '',
-				authenticationType: 'AzureMFA',
+				authenticationType: 'SqlLogin',
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts:47]: Why are you changing these? Given that this is going through an Azure account isn't AzureMFA the more reasonable default to use?

Comment (aaomidi) [reply extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts:47]: There are a few bugs with my approach here, I'm going to change this back to SqlLogin for May once my fixes are in.

Essentially this doesn't reuse your current connection and will keep prompting you for a password

File: extensions/azurecore/src/azureResource/commands.ts (hunk 1 above)
Comment (aaomidi) [reply extensions/azurecore/src/azureResource/commands.ts:234]: This change was suggested by @kburtram

Comment (kburtram) [reply extensions/azurecore/src/azureResource/commands.ts:234]: The change I was suggesting was for the bug where connections are being adding to the servers view as a byproduct of using them in Azure view.  By using, I mean expanding and authentication, etc.  Those actions should not result in a connection being generated in the Server view.  

If the user explicitly clicked that "Add Connection" button next to the server\database node then in that scenario a connection should be added to servers.

**Comment by kburtram** [comment]
Closing stale PRs

**Pull request closed without merging**